src/index.html

```diff
@@ -3,12 +3,32 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="image"><img src="images/logo.png" alt="moyo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"> <a class="nav__link is-active"href="#apple">Apple</a></li>
```

Comment:
Fix it from previous review with code style
```suggestion
          <li class="nav__item">
            <a class="nav__link is-active" href="#apple">Apple</a>
          </li>
```